pages/Network_APIs/Content_Production/CAMARA_APIs.md

```diff
@@ -1,60 +1,387 @@
 ---
 layout: default
-title: CAMARA APIs
+title: Analysis of CAMARA APIs
 parent: Content Production & Contribution
 grand_parent: Network APIs
-nav_order: 2
+nav_order: 4
 has_children: false
 ---
 
 {: .warning }
 This documentation is currently **under development and subject to change**. It reflects outcomes elaborated by 5G-MAG members as part of **WI: Network Exposure and APIs Supporting Media Services and Applications**
 We welcome and encourage contributions from the broader community. If you are interested in becoming a member of the 5G-MAG and actively participating in shaping this work, please contact the [Project Office](https://www.5g-mag.com/contact)
 
-# Requirements coverage by CAMARA APIs
+# Analysis of CAMARA APIs within the scope of Content Production & Contribution
 
-## Quality of Service
+## General information
 
- Requirement | CAMARA APIs  
- -- | --
-Ability to apply different QoS profiles to individual data flows coming from the same production device node | 
-Ability to separate media/data flows coming from the same production device node | 
-Delivery to endpoint (Application Media Server) may be identified by security/protocol/IP/port | 
-Ability to configure new or re-configure existing QoS profiles to be selected during runtime  | 
-Ability to select at runtime a QoS profile for a media flow | 
-Ability to receive ACK (success/fail) | 
+- CAMARA Repositories in GitHub: [Link](https://github.com/orgs/camaraproject/repositories?q=sort%3Aname-asc)
+- API Backlog: [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIbacklog.md)
+- Proposed new APIs: [Link](https://github.com/camaraproject/WorkingGroups/pulls)
 
-## Information monitoring, logging and/or Network assistance
+## List of CAMARA APIs
+TBC
```

Comment:
Potentially additional APIs:
* QOS Boking: 
  * https://github.com/camaraproject/QoSBooking
  * Yaml in PR: Only a withdrawn yaml: https://github.com/camaraproject/QoSBooking/pull/5

* https://github.com/camaraproject/NetworkSliceBooking
  * But no real progress since some time.